public/app/features/scenes/core/types.ts

```diff
@@ -13,9 +13,18 @@ export interface SceneObjectStatePlain {
   $variables?: SceneVariableSet;
 }
 
-export interface SceneLayoutChildState extends SceneObjectStatePlain {
+export interface SceneLayoutChildSize {
   size?: SceneObjectSize;
 }
+export interface SceneLayoutChildInteractions {
+  isDraggable?: boolean;
+  isResizable?: boolean;
```

Comment:
I would move these isDraggable and isResizable into same type as size props, think these belong to the same concept of LayoutChildOptions

Reply:
```
interface SceneLayoutChildOptions {
  x?: number;
  y?: number;
  width?: number; // grid units or separate w property that is in grid units? 
  height?: number;
  isDraggable?: boolean;
  isResizable?: boolean;
  ySizing?: // etc 
}
```